src/main/java/uk/gov/cshr/civilservant/config/OAuthProperties.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.cshr.civilservant.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+
+@Component
+@ConfigurationProperties(prefix = "oauth")
+public class OAuthProperties implements Serializable {
```

Comment:
INFO - I am guessing this needs to be extracted and can't be values anymore? Anyone knows the answer please give me a shout.

Reply:
@superafroman what is your take?

Reply:
Not sure what you mean..! The property values are automatically mapped into this object so we can pass this around rather than all the values individually.